comet/services/anime.py

```
@@ -1,89 +1,138 @@
+        self._fribb_url = "https://raw.githubusercontent.com/Fribb/anime-lists/refs/heads/master/anime-list-full.json"
 
     async def load_anime_mapping(self, session: aiohttp.ClientSession | None = None):
         if self.loaded:
+            return True
+
+        count = await database.fetch_val("SELECT COUNT(*) FROM anime_entries")
+        if count and count > 0:
+            await self._load_provider_ids()
+
+            if await self._is_cache_stale():
+                asyncio.create_task(self._refresh_from_remote(background=True))
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)